packages/s/scons/xmake.lua

```diff
@@ -0,0 +1,44 @@
+package("scons")
+    set_kind("binary")
+
```

Comment:
remove space line

Reply:
What do you mean by "space line"? The indent in line 2? Or should I add a line between package("scons") and set_kind("binary")?